DEVELOPMENT.md

```diff
@@ -62,7 +62,7 @@ of the sources _Github Source_, _AWS SQS Source_, _Camel Source_, _Kafka Source_
 with:
 
 ```
-ko apply -f contrib/<source_name>/
+ko apply -f <source_name>/config  # e.g. github/config
```

Comment:
above in the document, there is an outdated link to the `ko` page on github.

Can you update that, so it than points to https://github.com/google/ko   ?

Reply:
can you also add a statement to do the proper KO export,

like:  `export KO_DOCKER_REPO="docker.io/USERNAME"`

Reply:
Sure. Done